software/sama.yml

```
@@ -0,0 +1,12 @@
+name: SAMA
+website_url: https://samacloud.io
+source_code_url: https://github.com/SAMA-Communications/sama-server
+description: Next-Gen self-hosted chat server and clients.
+licenses:
+  - GPL-3.0
+  - Apache-2.0
+platforms:
+  - Nodejs
```

https://github.com/SAMA-Communications/sama-server?tab=readme-ov-file#deployment

```suggestion
  - Docker
  - Nodejs
```

---

updated